content/docs/pipelines/upgrade.md

```diff
@@ -119,92 +114,47 @@ your existing storage to retrieve the original data in the new cluster.
 **Note:** You must use command line deployment. You cannot reinstall
 Kubeflow Pipelines using the web interface.
 
-### Reinstalling Kubeflow Pipelines on GCP
-
-To reinstall Kubeflow Pipelines, follow the [command line deployment 
-instructions](/docs/gke/deploy/deploy-cli/), but note the following
-change in the procedure:
-
-1. Warning, when you do `kfctl apply` or `kfctl build`, you should use a 
-  different `${KF_NAME}` name from your existing `${KF_NAME}`. Otherwise, your 
-  data in existing PDs will be deleted during `kfctl apply`.
-
-1. **Before** running the following `apply` command:
-
-    ```
-    kfctl apply -V -f ${CONFIG_FILE}
-    ```
-
-    You should first:
-    * Edit `${KF_DIR}/gcp_config/storage-kubeflow.yaml` to skip creating new storages:
-
-      ```
-      ...
-      createPipelinePersistentStorage: false
-      ...
-      ```
-
-    * Edit the following files to specify the persistent disks created
-      in a previous deployment:
-
-      `${KF_DIR}/kustomize/minio/overlays/minioPd/params.env`
-      ```
-      ...
-      minioPd=[NAME-OF-ARTIFACT-STORAGE-DISK]
-      ...
-      ```
-
-      `${KF_DIR}/kustomize/mysql/overlays/mysqlPd/params.env`
-      ```
-      ...
-      mysqlPd=[NAME-OF-METADATA-STORAGE-DISK]
-      ...
-      ```
-
-1. Then run the `apply` command:
-
-    ```
-    kfctl apply -V -f ${CONFIG_FILE}
-    ``` 
-
-### Reinstalling Kubeflow in other environments (non-GCP) 
-
-The steps are the same as for any non-GCP installation, except that you
+The steps are the same as for a standard Kubeflow installation, except that you
 must use the same PV definitions as in your previous deployment to create the
 PV in the new cluster.
 
+To reinstall Kubeflow Pipelines:
```

Comment:
Question: the GCP reinstallation guide requires users to specify a different deployment name. Is that requirement GCP specific?

Reply:
The process is different for reinstalling a GCP deployment as opposed to non-GCP. I think it doesn't do any harm to use a different deployment name when reinstalling on non-GCP (it's probably best practice) but I'm not sure if it's essential. 

@Bobgy Can you shed some light here?

Reply:
For GCP, this is required, otherwise the new storage deployment will reuse the old one but clears existing persistent disks.

For other providers, I have never tried the process. Just for operational best practice (mentioned by @IronPan), it would be better to use a different name, because if any error occurs, it shouldn't break existing data.

Reply:
Thanks! I've updated the instructions to use a different deployment name.